index.ts

```diff
@@ -113,6 +113,19 @@ const addChildren = (
 	}
 };
 
+const reservedAttrs = new Set([
```

Comment:
Contexts goes on top, including a more specific variable name
```suggestion
// These attributes allow "false" as a valid value
// https://github.com/facebook/react/blob/a7d07ff24d8d3d925ab45f5eb9be42a62a345dad/packages/react-dom/src/shared/DOMProperty.js#L257-L289
const booleanishAttributes = new Set([
```